internal/helm/controller/reconcile.go

```diff
@@ -311,7 +312,28 @@ func (r HelmOperatorReconciler) Reconcile(ctx context.Context, request reconcile
 				"Chart value %q overridden to %q by operator's watches.yaml", k, v)
 		}
 		force := hasAnnotation(helmUpgradeForceAnnotation, o)
-		previousRelease, upgradedRelease, err := manager.UpgradeRelease(ctx, release.ForceUpgrade(force))
+		upgradeOptions := []release.UpgradeOption{
+			release.ForceUpgrade(force),
+		}
+
+		// this function allows the user to set the value for the annotation,
+		// "helm.sdk.operatorframework.io/rollback-force" to false. However,
+		// it does keep the default set to true
+		forceRollback := func() bool {
+			val, ok := o.GetAnnotations()[helmRollbackForceAnnotation]
+			if !ok {
+				return true
+			}
+			r, err := strconv.ParseBool(val)
+			if err != nil {
```

Comment:
Nit: Can we add a log stating that the annotation value could not be parsed, setting it to default instead of silently dropping the error.

Reply:
Certainly